app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  # Returns the full title on a per-page basis.
+  def full_title page_title
+    base_title = t(".helper")
```

Comment:
i18n dùng trong helper và model thì cần viết rõ cú pháp nhé em
```suggestion
    base_title = I18n.t(".helper")
```